app/components/find_interface/phase_banner/view.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module FindInterface
```

Comment:
Can we drop `Interface` ?

Reply:
Discussed offline - decided to keep as it makes things more clear

Reply:
Discussed offline - decided to keep this naming convention as it makes it more clear its not a part of publish.